erna/__init__.py

```diff
@@ -165,16 +193,20 @@ def load(
     data["filename"] = build_filename(data.NIGHT, data.RUNID)
     drs_data["filename"] = build_filename(drs_data.NIGHT, drs_data.RUNID)
 
-    # write path TODO: file ending? is everything in fz?
+    # write path TODO automaticly figures the out weather it is a gz or fz file
     data["path"] = data.apply(build_path, axis=1, path_to_data=path_to_data, extension='.fits.fz')
     drs_data["path"] = drs_data.apply(build_path, axis=1, path_to_data=path_to_data, extension='.drs.fits.gz')
 
+    drs_data = test_drs_path(drs_data, "path").groups[1]
+
     # reindex the drs table using the index of the data table.
     # There are always more data runs than drs run in the db.
     # hence missing rows have to be filled either forward or backwards
     earlier_drs_entries = drs_data.reindex(data.index, method="ffill")
+    earlier_drs_entries = earlier_drs_entries.fillna(axis="index", method="ffill")
```

Comment:
What does it do and why wasn't it necessary before?

Reply:
There are some corner cases with finding drs files that were not accounted for before.
If I remember it correctly, it basically failed if there were no drs files prior or post to the data files, which before never happend because technically there are always such files taken, but in my case the drs files were missing in the fhgfs and I check for this (thats the new part) and remove them before this. Because if I don't do this several data files will get drs files which are non existent.

Reply:
OK, for you max? I think it makes sense.